engine/src/main/java/org/destinationsol/game/screens/MainGameScreen.java

```diff
@@ -446,7 +448,7 @@ public void drawImages(UiDrawer uiDrawer, SolApplication solApplication) {
         myMoneyExcessTp.text = null;
 
         maybeDrawHeight(uiDrawer);
-        borderDrawer.draw(uiDrawer, solApplication);
+        borderDrawer.draw(uiDrawer, solApplication, solApplication.getContext());
```

Comment:
Here too. Is the SolApp context different? Maybe it should be... :thinking:

Reply:
It's the same as the SolGame context for now. I think #586 might makes some changes to that, but this should be independent of that.